### DISCUSSION TITLE
The pages build and deployment cannot deploy

### DISCUSSION THREAD

**Discussion by Kinki-Mochi** [community/community#12973] [Other Feature Feedback, Questions, & Ideas]
deployError: Request failed with status code 400
--
deployError: Request failed with status code 400
deployEnvironment URL '' is not a valid http(s) URL, so it will not be shown as a link in the workflow graph.

**Comment by counter-prop** [comment]
have same issue

**Reply by paurav11** [reply]
Did you find any solution to this issue?

**Comment by jfullstackdev** [comment]
it's already hosting but your `index.html` does not contain anything

![image](https://user-images.githubusercontent.com/47092464/181044316-887bf30e-c3ef-4d94-af5e-6c49b9412e3f.png)

![image](https://user-images.githubusercontent.com/47092464/181044176-5c9df7de-5d9f-4894-adb5-0f4a9f3bd04a.png)

**Comment by xgqfrms** [comment]
<img width="1307" alt="image" src="https://user-images.githubusercontent.com/7291672/200393206-54e82db8-bacd-4814-ad01-5653806467b2.png">
